examples/warpExposure.py

```diff
@@ -28,7 +28,15 @@
 import lsst.afw.image as afwImage
 import lsst.afw.math as afwMath
 import lsst.daf.base as dafBase
-import lsst.pex.logging
+import lsst.log
+
+
+def traceSetAt(name, number):
+    """Display messages with trace number <= NUMBER"""
+    for i in range(1, 6):
+        level = lsst.log.INFO if i > number else lsst.log.DEBUG
+        lsst.log.Log.getLogger('TRACE%d.%s' % (i, name)).setLevel(level)
+
```

Comment:
This utility function seems pretty generic.  Maybe we should move it to the `log` package itself so it can be used elsewhere?